ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -1284,6 +1284,51 @@ port_channel_interfaces:
 +       encapsulation: unmatched
 ```
 
+### Data model for BGP additional-paths in `router_bgp.address_family_evpn/_vrfs[].ipv4/_vrfs[].ipv6/_path_selection` has been changed
+
+In AVD 5.0.0, the data models for `router_bgp.vrfs[].address_family_ipv4.bgp.additional_paths`, `router_bgp.vrf[].address_family_ipv6.bgp.additional_paths`, `router_bgp.address_family_evpn.bgp.peer_groups[].additional_paths`, `router_bgp.address_family_evpn.bgp_additional_paths`, `router_bgp.address_family_path_selection.bgp.additional_paths`, `router_bgp.address_family_path_selection.neighbors[].additional_paths` have been changed. The key `additional_paths.send` is a string instead of dictionary.
```

Comment:
```suggestion
In AVD 5.0.0, the following data models have been changed and will require updates to the inventory:

- `router_bgp.vrfs[].address_family_ipv4.bgp.additional_paths`
- `router_bgp.vrf[].address_family_ipv6.bgp.additional_paths`
- `router_bgp.address_family_evpn.bgp.peer_groups[].additional_paths`
- `router_bgp.address_family_evpn.bgp_additional_paths`
- `router_bgp.address_family_path_selection.bgp.additional_paths`
- `router_bgp.address_family_path_selection.neighbors[].additional_paths`

The key `additional_paths.send` is a now a string instead of dictionary, and extra keys have been added for the various related arguments.
```